FinancialTracker/Helpers/ViewControllerFactory.swift

```diff
@@ -29,8 +28,6 @@ enum ViewControllerType {
     
     var value: String {
         switch self {
```

Comment:
Switch is not needed any more actually. Add rawValue String to the ViewControllerType and use it directly from there.